### DISCUSSION TITLE
macos 更新 0.18 后皮肤侯选高亮无法填充满整行

### DISCUSSION THREAD

**Discussion by Vergegung** [rime/home#1544] [Q&A] [answered]
如题，发现更新之后，可能是皮肤的一些默认设置要求更改了，所有的侯选字高亮区域填充满整行的，都变成四周有留白的了，请问如何设置回来，比如之前的win10主题，现在变成这样了
![CleanShot 2024-05-07 at 17 19 47@2x](https://github.com/rime/home/assets/38413818/303b7509-49f0-4e64-8168-0a82a8b68259)

**Comment by LEOYoon-Tsaw** [comment]
邊框調到零

**Reply by Vergegung** [reply]
边框能调大小？我只看到边框颜色调整😅

**Reply by LEOYoon-Tsaw** [reply, marked as answer]
border_height: 0
border_width: 0
另一個辦法是去掉邊框顏色

**Reply by Vergegung** [reply]
非常感谢，完美解决👍🏻，是不是更新之后规则变了，之前都没有问题

**Comment by Vergegung** [comment]
> border_height: 0 border_width: 0 另一個辦法是去掉邊框顏色

不好意思，再请教一个问题，新版本边框透明如何设置啊，😁

**Reply by LEOYoon-Tsaw** [reply]
用透明顏色就行了
